### PR TITLE
test:jest-unit-test-cases-transformer swift transformer file

### PR DESCRIPTION
**Pull request by kaviarasanopensource300**
# Description
test:jest-unit-test-cases-transformer swift transformer file
added the unit test for the swift transformer file
***required*** - *Description of the changes made in this Pull Request*

## Links

***required*** - *Links to resources related to this Pull Request*

### CODE REVIEW

**Review by bhanusengar — CHANGES_REQUESTED** [review]

File: packages/@momentum-design/builder/src/assets/builder/transformer/swift-transformer.unit.test.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import path from 'path';
+import type { Formats, SwiftFormat } from '../types';
+import Transformer from './transformer';
+import SwiftTransformer from './swift-transformer';
+import * as Utils from '../utils';
+
+describe('@momentum-design/builder - swift Transformer', () => {
+  let transformer: SwiftTransformer;
+  const FORMAT: Formats = { config: { fileName: 'MyFont', hbsPath: 'hpath' }, type: 'SWIFT' } as unknown as SwiftFormat;
```
[the remark this answers is not in the document]
Comment (kaviarasanopensource300) [reply packages/@momentum-design/builder/src/assets/builder/transformer/swift-transformer.unit.test.ts:9]: Yes. I have removed unknown

File: packages/@momentum-design/builder/src/assets/builder/transformer/swift-transformer.unit.test.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    });
+  });
+
+  describe('transformFilesAsync function', () => {
+    it('should return the correct data from the promise of transformFilesAsync() function', async () => {
+      transformer.inputFiles = [{
+        srcPath: 'font',
+        distPath: 'font',
+        // eslint-disable-next-line max-len, max-len
+        data: '{"0": {"name": "accessibility-bold","srcPath" : "testPath","codepoint" : "61697","codepointHexa" : "f101","unicode" : "testUnicode"}}',
```
[the remark this answers is not in the document]
Comment (kaviarasanopensource300) [reply packages/@momentum-design/builder/src/assets/builder/transformer/swift-transformer.unit.test.ts:34]: yes. I have stored in a variable.

**Review by bhanusengar — APPROVED** [review]
approved

**Review by gauravchauhan3030 — CHANGES_REQUESTED** [review]

File: packages/@momentum-design/builder/src/assets/builder/transformer/swift-transformer.unit.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaviarasanopensource300) [reply packages/@momentum-design/builder/src/assets/builder/transformer/swift-transformer.unit.test.ts]: Yes. I have removed.

[the remark this answers is not in the document]
Comment (kaviarasanopensource300) [reply packages/@momentum-design/builder/src/assets/builder/transformer/swift-transformer.unit.test.ts]: I have removed unknown.

**Pull request closed without merging**